docs/infrastructure/metrics-explorer.asciidoc

```diff
@@ -20,6 +20,7 @@ By default that is set to `@timestamp`.
 * The interval for the X Axis is set to `auto`.
 The bucket size is determined by the time range.
 * To use *Open in Visualize* you need access to the Visualize app.
+* To use *Create alert* you need alerting {kibana-ref}/alerting-getting-started.html#alerting-setup-prerequisites[setup].
```

Comment:
Sorry, another suggestion. `setup` is a noun. `set up` is a verb. Since this is the action of setting up, we need to use the verb. I would also recommend reorganizing the sentence like this:

```suggestion
* To use *Create alert* you need to {kibana-ref}/alerting-getting-started.html#alerting-setup-prerequisites[set up alerting].
```